### DISCUSSION TITLE
Optimize Examples Ordering

### DISCUSSION THREAD

**Discussion by Scienfitz** [emdgroup/baybe#64] [Documentation]
Back in Azure days we already had a sort of semi-manual ordering of the example topics. But that seems to have gone again, at the moment examples are ordered ... randomly?
![image](https://github.com/emdgroup/baybe/assets/17951239/075539fc-4461-4043-bec8-93c73079dd52)

Imo an ordering that follows ntural complexity makes more sense and users dont have to search for what is their first touch point. 

Suggestion:
- Basics
- Searchspaces
- Multi Target
- Everything else alphabetically

@AVHopp

**Comment by AVHopp** [comment]
The ordering was done by the headers that we used previously, which are gone now. It should be possible and rather easy to fix this as long as we can agree on an ordering. If the one one you suggest here also works for @AdrianSosic , I could probably adjust that rather quickly

**Reply by AdrianSosic** [reply]
"Simple" first sounds good!! The for the mid-range, however, it's hard to say what is "easier" and what "advanced". For instance, I personally think that "constraints" are more basic than "multi target", or at least they are closer to the "serach space" topic. But in the end, it depends also a bit on what is contained in the examples. What I mean: if "example A" (e.g. search space) already uses "Topic B" (e.g. constraints) but there is also a dedicated "example B" (constraints), then B really should come **before** A. You know what I mean.

Ultimately, the decision is yours :D

**Comment by Scienfitz** [comment]
closing as resolved / implemented
